### PR TITLE
Updated AboutUs

### PR DESCRIPTION
**Pull request by ChangUo79**

### CODE REVIEW

**Comment by hellodommy** [comment]
Looks ready to merge! 👍🏻Hope your picture shows up now.

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -20,7 +20,7 @@ Responsibilities: Scheduling and tracking, Deliverables and deadlines, *Model Co
 '''
 
 === Erwin Chan
-image::ChangUo79.png[width="150", align="left"]
+image::changuo79.png[width="150", align="left"]
```
Comment (jeunhoe) [inline docs/AboutUs.adoc:23]: Wow Erwin! Looking good! 😍

**Review by jeunhoe — COMMENTED** [review]
Hope your picture shows up on the tP showcase now.

File: docs/AboutUs.adoc (hunk 1 above)
Comment (shawnlsj97) [reply docs/AboutUs.adoc:23]: amazing work erwin, i shall merge this request!

**Pull request merged**